sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 
 ### Other Changes
 
+- Removed `resource_id`, please use `identity_config` instead.
+- Renamed argument name `get_assertion` to `func` for `ClientAssertionCredential`.
```

Comment:
The APIView has a suggestion to use `assertion` -- was `func` decided on during the review, and just not noted in the APIView?

Reply:
Thanks for pointing this out. We confirmed with our architects. We want to use "func" as the argument name.

Reply:
Thanks for the clarification! Everything LGTM then 🙂